src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php

```diff
@@ -103,14 +104,17 @@ public function testIsSelf(bool $expectedIsSelf, string $message, string $traceC
 
     public function providerIsIndirectUsesRightTrace(): array
     {
+        $reflexion = new \ReflectionClass(ClassLoader::class);
```

Comment:
french spotted :fr: :wink: 
`:%s/reflexion/reflection`

Reply:
I'm unmasked 😎. Thank you, it was just a test but it doesn't seem to work better.

Reply:
I think it works slightly better but there might be several problems.

Reply:
Ok, the second issue I have is that `/home/greg/dev/symfony/src/Symfony/Bridge/PhpUnit/vendor/bin/simple-phpunit` does not exist on my machine.

Reply:
It needs to exist because of the call to `realpath`